README.md

```diff
@@ -72,329 +93,235 @@ Mon Aug  5 17:52:22 2024
-|  GPU   GI   CI        PID   Type   Process name                              GPU Memory |
-|        ID   ID                                                               Usage      |
-|=========================================================================================|
-|  No running processes found                                                             |
+|  No MIG devices found                                                                   |
 +-----------------------------------------------------------------------------------------+
-
+...
 ```
 
-### To Deploy on the cluster
-
-**All in one command**
-
-make docker-build && make docker-push && make deploy
-
-Cross-platform or multi-arch images can be built and pushed using
-`make docker-buildx`. When using Docker as your container tool, make
-sure to create a builder instance. Refer to
-[Multi-platform images](https://docs.docker.com/build/building/multi-platform/)
-for documentation on building mutli-platform images with Docker.
-
-You can change the destination platform(s) by
-setting `PLATFORMS`, e.g.
+### Create instances of your solution**
+You can apply the samples (examples) from the `sample` directory:
 
-```sh
-PLATFORMS=linux/arm64,linux/amd64 make docker-buildx
-```
-
-**Build and push your image to the location specified by `IMG`:**
-
-```sh
-make docker-build docker-push IMG=<some-registry>/instaslice:tag
-```
-
-**NOTE:** This image ought to be published in the personal registry you specified.
-And it is required to have access to pull the image from the working environment.
-Make sure you have the proper permission to the registry if the above commands don’t work.
-
-**Install the CRDs into the cluster:**
-
-```sh
-make install
-```
-
-**Deploy the Manager to the cluster with the image specified by `IMG`:**
-
-```sh
-make deploy IMG=<some-registry>/instaslice:tag
+```console
+# kubectl apply -k samples/
 ```
 
-> **NOTE**: If you encounter RBAC errors, you may need to grant yourself cluster-admin
-privileges or be logged in as admin.
+**NOTE**: Ensure that the samples use the default values to test it out.
 
-**Create instances of your solution**
-You can apply the samples (examples) from the config/sample:
-
-```sh
-kubectl apply -k config/samples/
-```
+### Uninstall
 
->**NOTE**: Ensure that the samples has default values to test it out.
+1. Delete all running samples from the cluster:
 
-### To Uninstall
-**Delete the instances (CRs) from the cluster:**
-
-```sh
-kubectl delete -k config/samples/
+```console
+# kubectl delete -k samples/
 ```
 
-**Delete the APIs(CRDs) from the cluster:**
+2. Delete the CRDs:
 
-```sh
-make uninstall
+```console
+# make uninstall
 ```
 
-**UnDeploy the controller from the cluster:**
+3. Undeploy InstaSlice:
 
-```sh
-make undeploy
+```console
+# make undeploy
 ```
 
 ## Project Distribution
 
+**TODO**: Is this section still relevant? What does it do?
```

Comment:
@asm582 FYI, I didn't really know what to do with this section. Should I just leave it "as is"?

Reply:
This section was from the Kubebuilder template, our distribution strategy will change soon once we have CI/CD in place, so I vote for removing this section.